winpr/libwinpr/sspi/CredSSP/credssp.c

```diff
@@ -138,9 +137,7 @@ SECURITY_STATUS SEC_ENTRY credssp_QueryCredentialsAttributesA(PCredHandle phCred
 {
 	if (ulAttribute == SECPKG_CRED_ATTR_NAMES)
 	{
-		SSPI_CREDENTIALS* credentials;
-
-		credentials = (SSPI_CREDENTIALS*) sspi_SecureHandleGetLowerPointer(phCredential);
+		sspi_SecureHandleGetLowerPointer(phCredential);
```

Comment:
Well that's pretty much useless. The whole point of calling that function is to obtain the pointer, now you've removed the variable that was returned and kept the function alone. Agreed, the variable isn't used, worst case it could be commented out for the moment.